internal/shipment/handler.go

```diff
@@ -0,0 +1,116 @@
+package shipment
+
+import (
+	"github.com/gorilla/mux"
+	"net/http"
+	"notifications/pkg/api"
+	"notifications/pkg/jwt"
+	"strconv"
+)
+
+type APIShipmentHandler struct {
+	repo *shipmentRepo
+}
+
+func NewShipmentHandler(sr *shipmentRepo) *APIShipmentHandler {
+	return &APIShipmentHandler{
+		sr,
+	}
+}
+
+func RegisterRoute(sr *shipmentRepo, r *mux.Router) {
+	sh := NewShipmentHandler(sr)
+	r.HandleFunc("/shipment", sh.Create).Methods("POST")
+	r.HandleFunc("/shipments", sh.FindAll).Methods("GET")
+	r.HandleFunc("/shipment/{id}/", sh.FindOne).Methods("GET")
+	r.HandleFunc("/shipment/{id}", sh.Delete).Methods("DELETE")
+	r.HandleFunc("/shipment/{id}", sh.Update).Methods("PATCH")
+}
+
+func (handler *APIShipmentHandler) Create(w http.ResponseWriter, r *http.Request) {
+	tkn := jwt.TokenParse(r)
+	if tkn.Authorization == true {
+		shipment := Shipment{
```

Comment:
we should accept payload as application/json, not form value.
see this link https://www.sohamkamani.com/golang/json/